tests/xcm-transfer/shared.ts

```diff
@@ -31,18 +51,18 @@ export default function buildTest(tests: ReadonlyArray<TestType>) {
 
       let fromAccount = alice
       if ('fromAccount' in opt) {
-        fromAccount = opt.fromAccount(ctx)
+        fromAccount = (opt.fromAccount as any)(ctx)
       }
 
       let toAccount = alice
       if ('toAccount' in opt) {
-        toAccount = opt.toAccount(ctx)
+        toAccount = (opt.toAccount as any)(ctx)
       }
 
-      let precision = 3
-      if ('precision' in opt) {
-        precision = opt.precision
-      }
+      const precision = 3
+      // if ('precision' in opt) {
```

Comment:
why deprecate the `precision` option?

Reply:
sync with acala's e2e tests. I plan to replace this completely

Reply:
current setup is very hard to understand and keep up to date. every snapshot goes into one file. making it hard to understand which test failed, why failed, and why snapshot was update.
I plan to migrate every test into its file and write it like this https://github.com/AstarNetwork/e2e-tests/pull/26/files#diff-ede7f10848ecdee45d03f542c782549b4af14f83511e8cac1ec47fc41702a240